Gemfile

```diff
@@ -8,7 +8,7 @@ gem 'ejson'
 gem 'fog'
 gem 'fog-json'
 gem 'fog-xml'
-gem 'fog-dynect'
+gem 'fog-dynect', :git => 'https://github.com/Shopify/fog-dynect.git'
```

Comment:
It's documented on the [DynECT website](http://dyn.com/managed-dns/alias/). I think no one just added it to the library.